db/migrate/20201105014108_add_fill_status_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddFillStatusToItems < ActiveRecord::Migration[5.2]
+  def change
+    add_column :items, :fill_status, :string, default: "Unfulfilled"
```

Comment:
Does it make sense to put this here when the item_order is the one being fulfilled?